Sources/CreateAPI/Generator/Generator+Package.swift

```diff
@@ -8,19 +9,22 @@ extension Generator {
     }
 
     func makePackageFile(name: String) -> String {
-        let packages: String = [
-            #".package(url: "https://github.com/kean/Get", from: "1.0.2")"#,
-            isHTTPHeadersDependencyNeeded ? #".package(url: "https://github.com/CreateAPI/HTTPHeaders", from: "0.1.0")"# : nil,
-            isNaiveDateNeeded ? #".package(url: "https://github.com/CreateAPI/NaiveDate", from: "1.0.0")"# : nil,
-            isQueryEncoderNeeded ? #".package(url: "https://github.com/CreateAPI/URLQueryEncoder", from: "0.2.0")"# : nil
-        ].compactMap { $0 }.joined(separator: ", \n")
+        let allPackages = options.package.dependencies
+            .appending(.get)
+            .appending(.httpHeaders, if: isHTTPHeadersDependencyNeeded)
+            .appending(.naiveDate, if: isNaiveDateNeeded)
+            .appending(.urlQueryEncoder, if: isQueryEncoderNeeded)
+        
+        let packagesDeclaration: String = allPackages
+            .map(\.packageDeclaration)
+            .joined(separator: ",\n")
 
-        let dependencies: String = [
-            #".product(name: "Get", package: "Get")"#,
-            isHTTPHeadersDependencyNeeded ? #".product(name: "HTTPHeaders", package: "HTTPHeaders")"# : nil,
-            isNaiveDateNeeded ? #".product(name: "NaiveDate", package: "NaiveDate")"# : nil,
-            isQueryEncoderNeeded ? #".product(name: "URLQueryEncoder", package: "URLQueryEncoder")"# : nil
-        ].compactMap { $0 }.joined(separator: ", \n")
+        let dependenciesDeclaration = allPackages
+            .map(\.productDeclarations)
+            .reduce(Array<String>()) { partialResult, currentImports in
+                return partialResult.appending(contentsOf: currentImports)
+            }
+            .joined(separator: ",\n")
```

Comment:
I think you can also just `flatMap(_:)` the nested `[[String]]` array instead of having to use `reduce`

```suggestion
        let dependenciesDeclaration = allPackages
            .map(\.productDeclarations)
            .flatMap({ $0 })
            .joined(separator: ",\n")
```